src/Grpc.Net.Client/GrpcClient.cs

```diff
@@ -39,7 +39,33 @@ public static class GrpcClient
         /// <returns>A gRPC client.</returns>
         public static TClient Create<TClient>(HttpClient httpClient, ILoggerFactory? loggerFactory = null) where TClient : ClientBase
         {
-            return Cache<TClient>.Instance.Activator(new HttpClientCallInvoker(httpClient, loggerFactory));
+            if (httpClient == null)
+            {
+                throw new ArgumentNullException(nameof(httpClient));
+            }
+
+            return CreateCore<TClient>(new HttpClientCallInvoker(httpClient, loggerFactory));
+        }
+
+        /// <summary>
+        /// Creates a gRPC client using the specified <see cref="ChannelBase"/>.
+        /// </summary>
+        /// <typeparam name="TClient">The type of the gRPC client. This type will typically be defined using generated code from a *.proto file.</typeparam>
+        /// <param name="channel">The <see cref="ChannelBase"/>.</param>
+        /// <returns>A gRPC client.</returns>
+        public static TClient Create<TClient>(ChannelBase channel) where TClient : ClientBase
```

Comment:
Consider removing?

Codegen clients will have a constructor that takes ChannelBase. This method is slightly useful as it provides a simple fast way to use a generic argument to create a strongly typed client from a channel.

Reply:
This one seems a little more strenous though. I don't think this is simplifying anything?

`GrpcClient.Create<GreeterClient>(channelBase)` vs `new GreeterClient(channelBase)` doesn't seem like a significant improvement.